src/pages/FollowUpAnalysis/FollowUpAnalysisTable/ElementRow.js

```diff
@@ -17,13 +17,13 @@ const ElementRow = ({ element, onCheckboxToggle, onShowComment }) => (
         <TableRowColumn>{element.organisation}</TableRowColumn>
         <TableRowColumn>{element.period}</TableRowColumn>
         <TableRowColumn className={cssPageStyles.numericalRow}>
-            <FormattedNumber value={element.min} />
+            {element.min && <FormattedNumber value={element.min} />}
```

Comment:
The new API does not set `min` and `max` to 0 if they are not defined.

Reply:
Is it possible for element.min to be `0`? Because in that case this would hide the FormattedNumber input. Maybe we should make the check more explicit here if that's the case.

Reply:
I don't know if `element.min` can be 0 but I'll change this check to `typeof element.min === 'number' && ...` since that's both more explicit and safer. (and I'll update the `element.max` check too)